File: Programming Basics/Java/Conditional Statements/ToyShop.java
package com.company;

import java.util.Scanner;

public class ToyShop {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double excursionPrice = Double.parseDouble(scanner.nextLine());
        int puzzles = Integer.parseInt(scanner.nextLine());
        int dolls = Integer.parseInt(scanner.nextLine());
        int teddyBears = Integer.parseInt(scanner.nextLine());
        int minions = Integer.parseInt(scanner.nextLine());
        int trucks = Integer.parseInt(scanner.nextLine());
        double puzzlePrice = 2.60;
        double dollsPrice = 3;
        double teddysPrice = 4.10;
        double minionPrice = 8.20;
        double truckPrice = 2;
        double puzzleIncome = puzzles * puzzlePrice;
        double dollsIncome = dolls * dollsPrice;
        double teddyBearsIncome = teddyBears * teddysPrice;
        double minionsIncome = minions * minionPrice;
        double trucksIncome = trucks * truckPrice;
        double totalIncome = puzzleIncome + dollsIncome + teddyBearsIncome + minionsIncome + trucksIncome;
        double orders = puzzles + dolls + teddyBears + minions + trucks;
        if (orders >= 50) {
            totalIncome = (totalIncome * 0.75) * 0.9;
            if (totalIncome >= excursionPrice) {
                System.out.printf("Yes! %.2f lv left.", (totalIncome - excursionPrice));
            }else {
                System.out.printf("Not enough money! %.2f lv needed.", Math.abs(totalIncome - excursionPrice));
            }
        }else if (totalIncome >= excursionPrice) {
            System.out.printf("Yes! %.2f lv left.", ((totalIncome * 0.9) - excursionPrice));
        }else {
            System.out.printf("Not enough money! %.2f lv needed.", Math.abs((totalIncome * 0.9) - excursionPrice));
        }

    }
}

File: Programming Basics/JavaScript/Conditional Statements/Toy Shop.js
function ToyShop(input) {
    let excMoney = Number(input.shift());
    let quantityPuzzles = Number(input.shift());
    let quantityTalkingDolls = Number(input.shift());
    let quantityTeddyBears = Number(input.shift());
    let quantityMinions = Number(input.shift());
    let quantityTrucks = Number(input.shift());
    let PuzzlePrice = 2.60;
    let TalkingDollPrice = 3.00;
    let TeddyBearPrice = 4.10;
    let MinionPrice = 8.20;
    let TruckPrice = 2.00;
    let totalOrders = quantityMinions + quantityPuzzles + quantityTalkingDolls + quantityTeddyBears + quantityTrucks;
    let totalIncome = (quantityPuzzles * PuzzlePrice) + (quantityTalkingDolls * TalkingDollPrice) +
        (quantityTeddyBears * TeddyBearPrice) + (quantityMinions * MinionPrice) + (quantityTrucks * TruckPrice);
    if (totalOrders >= 50) {
        let totMin25 = totalIncome - (totalIncome * 0.25);
        let totMinMinRent = totMin25 - (totMin25 * 0.10);
        if (totMinMinRent >= excMoney) {
            console.log("Yes! " + (totMinMinRent - excMoney).toFixed(2) + " lv left.");
        }else {
            console.log("Not enough money! " + (excMoney - totMinMinRent).toFixed(2) + " lv needed.");
        }
    }else {
        let totMin25 = totalIncome
        let totMinMinRent = totMin25 - (totMin25 * 0.10);
        if (totMinMinRent >= excMoney) {
            console.log("Yes! " + (totMinMinRent - excMoney).toFixed(2) + " lv left.");
        }else {
            console.log("Not enough money! " + (excMoney - totMinMinRent).toFixed(2) + " lv needed.");
        }
    }

}
ToyShop([320,
    8,
    2,
    5,
    5,
    1    
    ])